src/alchemlyb/parsing/amber.py

```diff
@@ -174,6 +174,7 @@ def __init__(self):
         self.t0 = -1.0
         self.dt = -1.0
         self.T = -1.0
+        self.ntpr = -1
```

Comment:
Why do you set this to -1? 1 would make more sense.

Reply:
I completely agree with you, but I set it to -1 as all the other defaults were settled to -1 (also the other defaults are meaningless in my opinion, but I didn't want to move too much away from the current implementation with this simple fix...) I think that for now, it's ok like this for coherence, but then I'll certainly change all the defaults when I'll refactor the parser in a future PR (I'm leaving for one week's vacation right now :-) )!

Reply:
Thanks for the explanation. I do agree that consistency is good. However, I felt that the default value has to be something that is meaningful and can give reasonable results when a custom value is not provided. Writing a frame every -1 step will not give meaningful time. I do acknowledge that other default values are rubbish but for the new code, we want it to be something reasonable. What do you think @orbeckst ?

Reply:
I think that for this PR we could be consistent with the old behavior (also having a dt of -1 doesn't make much sense). I'm planning to refactor the whole parser with a future PR, anyway!

Reply:
Also, these values are not provided by the user, they are read directly from the AMBER output file, and they are always present in every proper AMBER output file, so they shouldn't have a default value at all in my opinion.

Reply:
I'm ok with the rubbish -1 here in this PR.

In general I'd say `None` is a better default but let's leave these discussion for another PR. @DrDomenicoMarson if you plan to address this point, please open a new issue so that it's not forgotten.